src/DatePicker/DatePickerDialog.js

```diff
@@ -136,7 +138,7 @@ class DatePickerDialog extends Component {
           repositionOnUpdate={true}
           open={open}
           onRequestClose={this.handleRequestClose}
-          style={styles.dialogBodyContent}
+          style={Object.assign(styles.dialogBodyContent, containerStyle)}
```

Comment:
The style is leaking to the body, I think it should be `Object.assign({}, styles.dialogBodyContent, containerStyle)` or `{...styles.dialogBodyContent ...containerStyle}`.